ancillary_codi_tables.sql

```diff
@@ -413,7 +433,9 @@ CREATE TABLE CDM.PRIVATE_ADDRESS_HISTORY
 	ADDRESS_STATE char(2) NULL,
 	ADDRESS_TYPE char(2) NOT NULL,
 	ADDRESS_PREFERRED char(2) NOT NULL,
-	ADDRESS_USE char(2) NULL,
+	ADDRESS_PERIOD_END date NULL,
+	ADDRESS_PERIOD_START date NULL,
+	ADDRESS_USE char(2) NOT NULL,

```

Comment:
`ADDRESS_USE` is nullable in the draft CODI DM IG 4.2

```suggestion
	ADDRESS_USE char(2) NULL,

```

Reply:
Looks like it is required in the PCOR data model though.